doc/faq.rst

```diff
@@ -26,6 +26,20 @@ See :ref:`contributing`. Before wanting to add a new algorithm, which is
 usually a major and lengthy undertaking, it is recommended to start with :ref:`known
 issues <easy_issues>`.
 
+What's the best way to ask questions about scikit-learn?
+--------------------------------------------------------------
+We of course want to help scikit-learn users. But for broader questions about
```

Comment:
I think this sentence should be merged with the next. "of course want to" sounds a bit odd to me, how is "Helping users is important to us, but for broader questions about..."